packages/app/src/app/components/CreateSandbox/Import/SuggestedRepositories.tsx

```diff
@@ -151,7 +162,40 @@ export const SuggestedRepositories = () => {
                         </Text>
                       ) : null}
                     </Stack>
-                    <StyledIndicator aria-hidden>Import</StyledIndicator>
+                    {isFree && repo.private ? (
+                      <StyledIndicator>
+                        <InteractiveOverlay.Item>
+                          <Link
+                            as={RouterLink}
+                            to="/pro"
+                            onClick={() => {
+                              modals.newSandboxModal.close();
+                            }}
```

Comment:
Should we track this action and/or add a source to `/pro`?

Reply:
Yep, I think we should. I'll add it :)